.circleci/config.yml

```diff
@@ -127,23 +127,10 @@ workflows:
             - check
             - test-integration
             - test-schema-nomis
-      - request-stage-approval:
```

Comment:
leave out deployment to stage so that one environment has the correct configuration